doc/quickstart/install.rst

```diff
@@ -196,20 +196,23 @@ To install from source for development, follow these instructions.
 -  Test that your installation was successful by running
    ``esmvaltool -h``.
 
-Pre-installed versions on HPC clusters
---------------------------------------
+Pre-installed versions on HPC clusters / other servers
+------------------------------------------------------
 
-You will find the tool available on HPC clusters and there will be no need to install it
-yourself if you are just running diagnostics:
+You will find the tool available on some HPC clusters, so there is no need to install it
+yourself if you are just running recipes:
```

Comment:
Would it be possible to use the same text as the ESMValTool version?
```suggestion
ESMValCore is available on the HPC clusters CEDA-JASMIN and DKRZ-Levante, and on the Met Office Linux
estate, so there is no need to install ESMValCore if you are just running recipes:
```

Reply:
Technically, the Core itself is not installed as a module. I'm not sure how conveniently one could use the Core functionalities through this "module load esmvaltool" but I could be wrong. I think that is why we kept it general here, e.g. "the tool is available..." but we could follow this suggestion if you like. I don't have a strong opinion on this.

Reply:
I have updated the suggestion so that it refers to ESMValTool. Although, would it be worth removing this section in favour of the ESMValTool section, given this section doesn't refer to ESMValCore?

Reply:
Your suggestion and removing this section from the Core documentation work both for me. Perhaps it's simpler and fine to keep it with the change that you're suggesting 👍

Reply:
I have a strong preference for removing this section, to reduce duplication and maintanence effort (this PR wouldn't have been needed then!) 😉 If you think this section is needed in the ESMValCore documentation, we could instead link to the ESMValTool documentation from this section?

Reply:
I agree, that would be the best thing to do 👍 We should just keep the title and the existing note, nothing else. I will suggest these new changes in a review. @mo-tgeddes, very sorry for dragging you into this 😄